hack/update_manifests.sh

```diff
@@ -20,4 +20,4 @@ else
 fi
 
 go run "$CONTROLLER_GEN_DIR"/cmd/controller-gen/main.go rbac
-go run "$CONTROLLER_GEN_DIR"/cmd/controller-gen/main.go crd
+go run "$CONTROLLER_GEN_DIR"/cmd/controller-gen/main.go crd --domain dev
```

Comment:
we need to do this change to be able to run `make manifests` otherwise the default domain is changed to k8s.io. We just manually changed it in generated CRDs but did not reflect it here